src/django_twc_project/README.md.jinja

```diff
@@ -3,58 +3,64 @@
 
 2. Run the setup script:
 
-   ```sh
-   just setup
-   ```
-
-   This will:
-   - Clean the repository of existing files, stop any Docker containers that may be running, delete Docker images and networks.
-   - Copy the `.env.example` file to `.env` if it does not already exist.
-   - Create a virtual environment at the `$VIRTUAL_ENV` directory (default `.venv`).
-   - Install `uv` and `pre-commit` Python tools.
-   - Build the Docker images for the development environment.
-   - Install Python and Node.js dependencies.
-   - Start the Docker Compose stack.
-   - Migrate the database.
-   - Create a superuser for the Django admin interface with the username `admin` and the password `admin`.
-   - Run the test suite and generate code coverage.
-   - Run the `mypy` static type checker.
-   - Lint the repository.
-   - Stop the Docker Compose stack.
+    ```sh
+    just setup
+    ```
+
+    This will:
+
+    - Clean the repository of existing files, stop any Docker containers that may be running, delete Docker images and networks.
+    - Copy the `.env.example` file to `.env` if it does not already exist.
+    - Create a virtual environment at the `$VIRTUAL_ENV` directory (default `.venv`).
+    - Install `uv` and `pre-commit` Python tools.
+    - Build the Docker images for the development environment.
+    - Install Python and Node.js dependencies.
+    - Start the Docker Compose stack.
+    - Migrate the database.
+    - Create a superuser for the Django admin interface with the username `admin` and the password `admin`.
+    - Run the test suite and generate code coverage.
+    - Run the `mypy` static type checker.
+    - Lint the repository.
+    - Stop the Docker Compose stack.
 
 3. After the setup script has successfully completed, you can start your development Docker Compose stack by running either of these commands:
 
-   ```sh
-   # to start the containers detached in the background
-   just docker start
-   # to start the containers in the foreground with the logs continuously streaming in stdout
-   just docker up
-   ```
+    To start the containers in the background:
 
-   After starting, you should be able to access the development server at [http://localhost:8000](http://localhost:8000).
+    ```sh
+    just docker start
+    ```
 
-   By default, the development server will run on port 8000. If you need to change the port, you can do so by setting the `DJANGO_PORT` environment variable in the `.env` file and then restarting the development server.
+    To start the containers in the foreground with the logs continuously streaming in stdout
+
+    ```sh
+    just docker up
+    ```
+
+    After starting, you should be able to access the development server at [http://localhost:8000](http://localhost:8000).
+
+    By default, the development server will run on port 8000. If you need to change the port, you can do so by setting the `DJANGO_PORT` environment variable in the `.env` file and then restarting the development server.
```

Comment:
This is all the same, it's just split up a little bit for formatting purposes. (I was having trouble getting the list, code, and paragraph to render appropriately in mkdocs, and splitting it up seemed to help.)